code/modules/VR/vr_sleeper.dm

```diff
@@ -26,17 +26,17 @@
 	build_spawnpoints()
 	update_icon()
 
-/obj/machinery/vr_sleeper/attack_hand(mob/user)
-	. = ..()
-	if(.)
+/obj/machinery/vr_sleeper/attackby(obj/item/I, mob/user, params)
```

Comment:
isn't this just a copypaste of the machinery one? also, you need to return TRUE if you handled it, otherwise the attack chain continues and you screwdrivers etc bash the machine.

Reply:
Nope, and the chain doesn't continue in testing. Also looking at other machinery they don't return true either.